### PR TITLE
[SASS-6226] Limit BigDecimal form inputs to 2 decimal places

### PR DESCRIPTION
**Pull request by Jake-Raffe**

### CODE REVIEW

**Comment by lukastymo** [comment]
Great PR, more lines removed than added!

File: app/forms/mappings/Formatters.scala [hunk 1]
```diff
@@ -102,6 +102,33 @@ trait Formatters {
         baseFormatter.unbind(key, value.toString)
     }
 
+  private[mappings] def currencyFormatter(requiredKey: String, nonNumericKey: String, args: Seq[String] = Seq.empty): Formatter[BigDecimal] =
```
Comment (lukastymo) [inline app/forms/mappings/Formatters.scala:105]: should we create FormattersSpec and test currencyFormatter in isolation. I'm interested in more edge cases scenarios. I think right now we have just general controller tests. But I'm thinking of more cases just for this function, it should be easy testable.

**Review by lukastymo — DISMISSED** [review]

File: app/forms/mappings/Formatters.scala (hunk 1 above)
Comment (Jake-Raffe) [reply app/forms/mappings/Formatters.scala:106]: Pushed up now

**Review by lukastymo — APPROVED** [review]

**Review by conniebernardin — APPROVED** [review]
